.github/workflows/slapr.yml

```
@@ -14,6 +14,10 @@ jobs:
         channel_variables:
           - SLACK_AGENT_PLATFORM_ID
           - SLACK_AGENT_PLATFORM_REVIEWS_ID
+          - SLACK_OPENTELEMETRY_ID
+          - SLACK_AGENT_METRICS_LOGS_ID
+          - SLACK_CONTAINTERS_INTEGRATION_ID
```

```suggestion
          - SLACK_CONTAINERS_INTEGRATION_ID
```

---

```suggestion
          - SLACK_CONTAINER_INTEGRATIONS_ID
```
To match with the channel name